bedrock/firefox/templates/firefox/challenge-the-default/landing-base.html

```diff
@@ -362,12 +362,14 @@ <h3>{{ cred_private_title }}</h3>
         </summary>
         <p>{{ cred_private_body }}</p>
       </details>
+    {% if cred_ethical_title %}
```

Comment:
I had no copy for this bit for Polish, not sure if it was intentionally omitted or just missed in translation.